app/controllers/concerns/metrics_utils.rb

```diff
@@ -0,0 +1,61 @@
+require "active_support/concern"
+
+module MetricsUtils
```

Comment:
new controller concern to house shared methods used by both the `episode_metrics_controller` and `podcast_metrics_controller`.